gsy_framework/constants_limits.py

```diff
@@ -181,6 +183,20 @@ class WindSettings:
         FINAL_SELLING_RATE = 0
         MAX_WIND_TURBINE_OUTPUT_W = 160
 
+    class HeatPumpSettings:
+        """Default values for the heat pump."""
+
+        MAX_POWER_RATING_KW = 3
+        AVERAGE_CONSUMPTION_KW = 0.5
+        MIN_TEMP_C = 50
+        MAX_TEMP_C = 65
+        INIT_TEMP_C = 50
+        EXT_TEMP_C = 10
+        TANK_VOL_L = 50
+        TYPE = HeatPumpSourceType.AIR_SOURCE.value
```

Comment:
In order to make it less generic, I think that the constant variable would be better named as SOURCE_TYPE, and the enums as AIR and GROUND respectively.